app/commands/TransferWinnersCommand.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputArgument;
+
+class TransferWinnersCommand extends Command {
+
+	/**
+	 * The console command name.
+	 *
+	 * @var string
+	 */
+	protected $name = 'transfer:winners';
+
+	/**
+	 * The console command description.
+	 *
+	 * @var string
+	 */
+	protected $description = 'Collect winners data and transfer to winners table.';
+
+	/**
+	 * Create a new command instance.
+	 *
+	 * @return void
+	 */
+	public function __construct()
+	{
+		parent::__construct();
+	}
+
+	/**
+	 * Execute the console command.
+	 *
+	 * @return mixed
+	 */
+	public function fire()
+	{
+    try {
+      $transfer = Winner::transferWinners();
+      $this->info('Transfer completed, have a scholarly beer!');
```

Comment:
I initially added the :beer: emoji in the message, but then realized if it's running on linux on server, I might break the server if it doesn't support emoji :cry: